CMakeLists.txt

```diff
@@ -25,11 +25,6 @@ endif()
 
 include(${PROJECT_SOURCE_DIR}/toolchains/utils.cmake)
 
-# cmake output configuration
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
```

Comment:
If we remove that from here we need to update the readme in several places. Could this be android specific if only useful here?